ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -198,10 +198,10 @@
           + "JSON; this helps proactively determine if the connection has been terminated in "
           + "order to avoid keeping the created streams job alive longer than necessary";
 
-  static final String DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_CONFIG =
+  public static final String DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_CONFIG =
       KSQL_CONFIG_PREFIX + "server.command.response.timeout.ms";
 
-  private static final String DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_DOC =
+  protected static final String DISTRIBUTED_COMMAND_RESPONSE_TIMEOUT_MS_DOC =
```

Comment:
any reason for this change?

Reply:
I saw that a lot of the other configs that were public had protected docs like `ACCESS_CONTROL_ALLOW_HEADERS,` so I just followed that

Reply:
👍